events_examples/benchling_request_to_slack/lambda_function.py

```diff
@@ -0,0 +1,60 @@
+# lambda_function.py
+
+import json
+import os
+import urllib3
+
+
+def lambda_handler(event, context):
+    http = urllib3.PoolManager()
+    assignee_handles = []
+    for assignee_handles in event["detail"]["request"]["assignees"]:
+        assignee_handles.append("@" + str(assignee_handles["user"]["handle"]))
```

Comment:
Can you rename one of the `assignee_handles`? It is a bit confusing that they all have the same name.

Reply:
Good call! Will updated that!